x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/connector_configuration_form.tsx

```diff
@@ -56,21 +57,41 @@ export const ConnectorConfigurationForm = () => {
       component="form"
     >
       {localConfigView.map((configEntry) => {
-        const { depends_on: dependencies, key, label } = configEntry;
+        const {
+          default_value: defaultValue,
+          depends_on: dependencies,
+          key,
+          display,
+          label,
+          tooltip,
+        } = configEntry;
+        // toggle label goes next to the element, not in the row
         const hasDependencies = dependencies.length > 0;
+        const helpText = defaultValue
+          ? `If left empty, the default value ${defaultValue} will be used.`
+          : '';
+
+        const rowLabel =
+          display !== 'toggle' ? (
```

Comment:
The fact that we're doing a naked string check here suggests to me that we should type`display` as an enum (or limited number of string values) while we're at it.

Reply:
Changed to enum